packages/report-flaky-tests/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/report-flaky-tests",
+	"version": "0.0.0",
+	"private": true,
+	"description": "A GitHub action to report flaky E2E tests to GitHub issues.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"E2E",
+		"flaky",
+		"GitHub Actions"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/report-flaky-tests/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/report-flaky-tests"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=14",
+		"npm": ">=6.9"
+	},
+	"main": "build/index.js",
```

Comment:
Should we add `"types": "build-types"`?

Reply:
Oops, I forgot about that. However, I don't think we really need to output the types for this package though. It's not really designed to be consumed by consumer code, but just for the GH workflows 😅.